src/application-error-boundary/ApplicationErrorBoundary.jsx

```diff
@@ -42,6 +43,7 @@ class ApplicationErrorBoundary extends React.Component {
      * When an exception occurs while rendering children, getDerivedStateFromError gets executed. The error
      * is returned here and stored within the ApplicationErrorBoundary's state, triggering an update.
      */
+    logger.error(error);
```

Comment:
I'd move this down into componentDidCatch below. Feels a little weird having side effects in this function.